manifests/audisp.pp

```diff
@@ -103,7 +103,7 @@
       group   => $config_group,
       mode    => $config_mode,
       content => epp('auditd/audisp.conf.epp', {
-        config => $config,
```

Comment:
I think this linting warning is a bug? https://github.com/voxpupuli/puppet-lint-strict_indent-check/issues/21

If that is the case (feel free to correct me) I think I would prefer this format:

```puppet
content => epp('auditd/audisp.conf.epp',
  {
    config => $config,
  }
),
```